prime-router/docs/onboarding-users/receivers.md

```diff
@@ -1,12 +1,307 @@
+```
+* In the above example, the jurisdictional filter searches the `ordering_facility_state` field in the report for anything 
+that matches the code LT.
+* Filters can be applied to the organization or receiver. For more information on filters see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how-to-use-filters.md)
+* In addition, there is the translation section, which specifies the output format that will be sent to the receiver. 
+Currently, we have three formats available:
+    - HL7
+    - CSV
+    - FHIR
+* The quality filters verify that test results have the minimum fields required by most public health jurisdictions. 
+* Those fields are made adjustable to be customized per jurisdictional specifications.
+* These filters are applied by default to the receiver and set in code.
+* They do not need to be added to the receiver setting.
+* If a custom rule is added to the quality filter for a receiver, all default quality filters will be ignored. The default filter includes 
+these requirements:
+- The following fields all have a value:
+     - Patient First Name
+     - Patient Last Name
+     - Patient DOB
+     - Test Result
+     - Specimen Type
+     - Specimen ID/Accession Number
+     - Device Identifier
+    
+- At least one of the following fields has a value:
+     - Patient Street Address
+     - Patient Zip Code
+     - Patient Phone
+     - Patient Email
+    
+- At least one of the following date fields has a valid date:
+     - Test Ordered Date
+     - Specimen Collection Date
+     - Test Result Date
+    
+- The following fields are 10 characters in length:
+     - Testing Lab CLIA
+     - Ordering Facility CLIA
+    
+- Processing Code does not equal T nor D
+
+- Only test results that pass all the above requirements will be transferred to the jurisdiction. There are two options 
+for jurisdictions that want all results reported.
+- Use AllowAll() to bypass all quality filters 
+- Create a secondary feed with the reverseQualityFilter() set to true. This will only allow results that fail the 
+quality filters listed above.
+
+The mechanism for how each record is translated is laid out in the schema, which is discussed in the next section.
+
+### 2. Set up a new schema
+*NOTE - IF YOU ARE WORKING ON AN HL7 RECEIVER, YOU DO NOT NEED TO CREATE A NEW SCHEMA.*
+
+* By default, any HL7 receiver will use the universal or covid schema and you do not need to create a schema
+specific to your receiver.
+* If they are going to receive a CSV or FHIR file you *MUST* create a schema. In lieu
+of a schema, we use the `TranslationConfig` to set default values and control HL7 processing. 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/playbooks/how_to_use_translation_configuration_features.md)
+* For additional information on creating a schema see: 
+(https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/how-to-onboard-a-sender.md)
+
```

Comment:
This whole section is incorrect. We can replace it with this:
* In the UP for HL7 v2 we have to check what HL7 message type they want to receive data in. We support ADT_A01, OML_O21 and ORU_R01. Depending on the message type we can set `translationSchema` to the respective schema.
* If the receiver wants specific receiver transforms that are not supported by the `translation` settings a schema can be created for them. More information on how to mange translation schemas can be found here (https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/universal-pipeline/translate.md)

Reply:
changed